packages/mobile/src/components/TokenTotalLineItem.tsx

```diff
@@ -44,11 +49,17 @@ export default function TokenTotalLineItem({
           <Text style={styles.exchangeRate} testID="TotalLineItem/ExchangeRate">
             {tokenInfo?.symbol}
             {' @ '}
-            <TokenDisplay
-              amount={new BigNumber(1)}
-              tokenAddress={tokenAddress}
-              showLocalAmount={true}
-            />
+            {localAmount?.exchangeRate ? (
+              `${
+                LocalCurrencySymbol[localAmount.currencyCode as LocalCurrencyCode]
+              }${formatValueToDisplay(new BigNumber(localAmount.exchangeRate))}`
+            ) : (
+              <TokenDisplay
+                amount={new BigNumber(1)}
+                tokenAddress={tokenAddress}
+                showLocalAmount={true}
+              />
+            )}
```

Comment:
We should avoid concatenating strings this way as we can't properly translate them.
Can we add a new translation key instead?

Reply:
Isn't this the same as what we're doing on `TokenDisplay` and `CurrencyDisplay`? We are doing string concatenation there as well. Is there any language where showing symbol and then value is not the convention?

Reply:
There are 2 concerns.

Here I meant a key for `<Something> @ <Rate>` like we have in https://github.com/valora-inc/wallet/blob/c1e0fae65108e064ce313f56ae85cbba84b1810f/packages/mobile/src/components/TotalLineItem.tsx#L66-L78

And the 2nd concern (which doesn't need to be addressed here) is the rules for formatting monetary amounts which we are not doing correctly right now as we assume the symbol is always first and then the value.
For instance in France the convention is to show it like `1.000 €`.
And there are more exotic formats like for Cape Verde Escudo (CVE) where the decimal separator is using the $ sign: `20$50` indicates 20 Escudos and 50 cents.

Reply:
Got it, added the translation key for now